### DISCUSSION TITLE
Trust Wallet button not working in Trust Wallet browser

### DISCUSSION THREAD

**Discussion by ls-jordan** [rainbow-me/rainbowkit#1252] [Q&A] [answered]
The Trust Wallet button does not work on TW dApp browser (does nothing when pressed), but Wallet Connect and Metamask buttons do. Additionally, the browser wallet option no longer appears when it previously did. I am using 0.12.14.

**Comment by DanielSinclair** [comment, marked as answer]
This should be resolved in more recent versions of RainbowKit.
